src/codelabs/iterables/index.md

```diff
@@ -0,0 +1,571 @@
+---
+title: "Iterable Collections in Dart"
+description: Learn about and practice using Iterables in Dart!
+---
+
+This codelab teaches you how to use collections that implement the 
+[Iterable][iterable class] class, for example [List][list class] and
+[Set][set class]. Using the embedded DartPad editors, you can test your
+knowledge by running example code and completing exercises.
+
+To get the most out of this codelab, you should have the basic knowledge
+of the [Dart syntax](/samples).
+
+This codelab covers the following material:
+
+* How to access elements of an Iterable.
+* How to check if the elements of an Iterable match a condition.
+* How to filter the contents of an Iterable.
+* How to map the contents of an Iterable to a different value.
+
+Estimated time to complete this codelab: 60 minutes.
+
+{{site.alert.note}}
+  This page uses embedded DartPads to display examples and exercises.
+  {% include dartpads-embedded-troubleshooting.md %}
+{{site.alert.end}}
+
+## Why do you need collections
+
+Collections are classes that represented a group of objects, also called "elements".
+
+A collection can be empty, or contain many elements in it. Depending on its purpose,
+a collection will follow a different data structure and have different implementations.
+These are some of the most common collection types:
+
+* [List][list class]: Used to access elements by their index.
+* [Set][set class]: Used to contain elements that can only occur once.
+* [Map][map class]: Used to access elements using a key.
+
+In Dart, both `List` and `Set` are `Iterable`, meaning that they share the same methods as the 
+`Iterable` class. 
+
+## What is an Iterable?
+
+An `Iterable` is a collection of elements that can be accessed sequentially.
+
+`Iterable`, in Dart, is an abstract class, meaning that you cannot instantiate it directly.
+However you can create a new `Iterable` by creating a new `List` or a `Set`.
+ 
+This example shows a `List` of `int`, which is also an `Iterable` of `int`:
+
+{% prettify dart %}
+    Iterable<int> iterable = [1, 2, 3];
+{% endprettify %}
+
+The difference with a `List`, is that with the `Iterable` you cannot 
+access elements using their index.
```

Comment:
It you can't assign, then it sounds like instead of "access" as a verb, we should say "read"/"get"/"use". (I wasn't going to get into this before the copy-edit phase, but "access" as a verb is a red flag — imprecise at best: https://developers.devsite.corp.google.com/style/word-list#letter-A)

Reply:
Thanks, I am rewriting that section. I don't have access to the linked word-list document but I understand the idea that access can be confusing so I am using read instead. i.e. reading elements rather than accessing elements.